cli/lint.rs

```diff
@@ -105,9 +105,9 @@ pub async fn lint_files(
 pub fn print_rules_list() {
   let lint_rules = rules::get_recommended_rules();
 
-  println!("Available rules:");
+  info!("Available rules:");
   for rule in lint_rules {
-    println!(" - {}", rule.code());
+    info!(" - {}", rule.code());
```

Comment:
This change makes no sense - `deno lint --unstable --rules` is only used to print out a list of rules, if we apply `info!()` here then you might not get output at all

Reply:
No, it does work. `deno lint --unstable --rules` prints the rules, except that it goes to stderr (not stdout).
If we add `--rules` and `--quiet` options at the same time, then we will get nothing.

![image](https://user-images.githubusercontent.com/23649474/93334659-40724180-f860-11ea-904f-efa8530f3e75.png)

Reply:
@magurotuna that's exactly my problem `--rules --quiet` makes no sense - in that case rules should still be printed - that's the only use case for this flag.

Reply:
Oh, I got it. You're right. I'll fix it to use `println!` here.

Reply:
Done!